apps/create-fuels-counter-guide/src/app/page.tsx

```diff
@@ -49,7 +49,11 @@ export default function Home() {
 
     if (walletBalance?.eq(0)) {
       return toast.error(
-        "Your wallet does not have enough funds. Please click the 'Top-up Wallet' button in the top right corner, or use the local faucet.",
+        <span>
+          Your wallet does not have enough funds. Please click the 'Top-up
```

Comment:
I think we can just include the link.

```suggestion
Your wallet does not have enough funds. Please top it up with the {" "}
          <Link href={FAUCET_LINK} target='_blank'>faucet.</Link>
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2798/commits/97b13d9951e7d282380ac61de620dc115b1b0ce2